tests/tkupchyn/homework_05/test_popular_words.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from tkupchyn.homework_05.popular_words import popular_words
+
+text_1 = "\nWhen I was One\nI had just begun\nWhen I was Two\nI was nearly new\n"
+text_2 = ("At the input of your function are given 2 arguments: the text and the array of words the popularity of "
+          "which you need to determine.")
```

Comment:
for testing you do not need to create big datasets, just depict important cases:

```
"foo\nbuzz"
"foo foo"
"Foo foo"
"foo: Buzz buzz foo"

```
etc.
depends on what you want to test